src/pages/Reviews.js

```diff
@@ -120,14 +98,6 @@ const Reviews = ({ productId }) => {
         <p>
           <em>Max. one review per product</em>
         </p>
-        <label htmlFor="product">Product:</label>
-        <div>
-        <input
-          type="text"
-          name="product"
-          value={formData.product}
-          onChange={handleInputChange}
-        /></div>
```

Comment:
We don't want the user to change the product ID that they're submitting the review for here, so let's remove that input element